crates/codegen/parser/src/rust_lib_code_generator.rs

```diff
@@ -92,11 +102,22 @@ impl CodeGenerator {
                             {version_flag_declarations}
                         }}
 
+                        #[derive(thiserror::Error, Debug)]
+                        pub enum Error {{
+                            #[error(\"Invalid version '{{0}}'.\")]
```

Comment:
Suggestion for both rust+typescript:

```suggestion
                            #[error(\"Invalid Solidity language version '{{0}}'.\")]
```

Reply:
Agreed. But is it possible to use/inject `grammar.title` there?

Reply:
Or perhaps just `Invalid language version`. I want to make sure errors distinguish slang/compiler version from the language version it is compiling.